x-pack/plugins/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -389,6 +401,50 @@ export class ClusterClientAdapter<TDoc extends { body: AliasAny; index: string }
     }
   }
 
+  public async queryEventsWithAuthFilter(
```

Comment:
I'm wondering if we should reduce some of the code duplication between this and `queryEventsBySavedObjects`, or maybe we can modify `queryEventsBySavedObjects` to support the new auth filters? Curious to hear your thoughts

Reply:
I don't want to modify the existing `queryEventsBySavedObjects` functionality too much, I'm not sure if it'll modify expected error behaviors in unpredictable places.